server/api/api.py

```diff
@@ -65,6 +65,7 @@
     orders.router,
     prefix="/orders",
     tags=["orders"],
+    dependencies=[Depends(auth_required)],
```

Comment:
I added the auth to all the order routes, not specific ones like before. Let me know if this is not OK.